client/src/hooks/useBigNumberJson.tsx

```diff
@@ -0,0 +1,18 @@
+import BN from "bn.js";
+import { useEffect } from "react";
+
+export const useBigNumberJson = () => {
```

Comment:
Maybe this can only be used in the parent component instead of every child component?

Reply:
Agreed, I've moved it to just Test